exchange_calendars/calendar_utils.py

```diff
@@ -389,3 +439,17 @@ def clear_calendars(self):
 register_calendar_type = global_calendar_dispatcher.register_calendar_type
 register_calendar_alias = global_calendar_dispatcher.register_calendar_alias
 resolve_alias = global_calendar_dispatcher.resolve_alias
+aliases_to_names = global_calendar_dispatcher.aliases_to_names
+names_to_aliases = global_calendar_dispatcher.names_to_aliases
+
+
+def get_exchanges_info() -> pd.DataFrame:
+    """Retrieve information on exchanges for which calendars available.
+
+    Returns
+    -------
+    pd.DataFrame
+        Information on exchanges for which calendars are maintained by
+        exchange_calendars (calendar names as column 'ISO Code').
+    """
+    return pd.read_html("https://github.com/gerrymanoim/exchange_calendars")[0]
```

Comment:
Is this getting the README as a `DataFrame`?

My strong preference is that `get_echanges_info` should reflect the calendars available to your local installation of `exchange_calendars`. To that end, this should either reflect a file we hardcode, or be built dynamically by looking at which calendars have been registered (and maybe we need to go add meta info to each calendar to generate this). I don't have a strong view about which way this gets done.  Happy to do this as a separate PR. 

(I'm also not entirely sure that programmatic access to that table is useful? But open to ideas.)

Reply:
Thanks for the feedback.

> Is this getting the README as a `DataFrame`?

Yes.

I use `exchange_calendars` exclusively with the default calendars and am probably somewhat insensitive to other usage cases where calendars are being actively (de)registered. I use `get_exchange_info` within a notebook simply as a quick way to cross reference default calendars with exchanges.

I've moved the branch back to exclude the related commit.